lib/modules/apostrophe-login/index.js

```diff
@@ -311,6 +313,37 @@ module.exports = {
       });
     };
 
+    self.disableIfInactive = function(err, user, callback) {
+      if (self.apos.users.options.disableInactiveAccounts) {
+        var usersOptions = self.apos.users.options.disableInactiveAccounts;
+
+        if (user.lastLogin) {
+          // by default, "admin" users are not disabled
+          var isUserWhitelisted = user._groups.some(function(group) {
+            return usersOptions.whitelistGroups.includes(group.title);
+          });
+
+          if (!isUserWhitelisted) {
+            var lastLogin = moment(user.lastLogin);
+            var now = moment();
+            var diff = lastLogin.diff(now, 'days');
+
+            if (diff >= usersOptions.inactivityDuration) {
+              var req = self.apos.tasks.getReq();
+              user = Object.assign({}, user, { disabled: true, trash: false, lastLogin: '' });
+              return self.apos.users.update(req, user)
+                .then(function() {
+                  return callback('Account disabled due to inactivity. Please, refer to the administrator of the site for assistance.', false);
+                }).catch(function(err) {
+                  self.apos.utils.error(err);
```

Comment:
Report this error to the original callback too, otherwise it looks like success to the user.

Reply:
Done